rdm/render.py

```diff
@@ -34,6 +34,34 @@ def join_to(foreign_keys, table, primary_key='id'):
     return joined
 
 
+def md_indent(snippet, header_shift=0):
+    lines = snippet.split('\n')
+    i = 0
+    in_code = False
+    if header_shift < 0:
```

Comment:
It seems like the two branches of this if statement are mostly redundant. I'd see if I could find a way to avoid the redundancy. I've got a few ideas of how, but I'd like to see what you can come up with.